src/main/java/net/dv8tion/jda/core/requests/WebSocketClient.java

```diff
@@ -864,5 +887,67 @@ protected String handleInternally(JSONObject content)
             });
         }
     }
+
+    public enum CloseCode
```

Comment:
This enum should be pulled out of WebSocketClient to it's own class in `requests` and provided with javadoc.